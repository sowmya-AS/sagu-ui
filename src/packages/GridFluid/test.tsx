import React from 'react'
import { render } from '../../utils/testUtils'

import GridFluid from '.'

describe('<GridFluid />', () => {
  it('should render GridFluid correctly with children', () => {
    const { container } = render(
      <GridFluid>
        <div>GridFluid</div>
      </GridFluid>
    )
    expect(container).toMatchSnapshot()
  })
})
