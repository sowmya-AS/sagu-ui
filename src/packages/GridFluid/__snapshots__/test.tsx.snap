// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridFluid /> should render GridFluid correctly with children 1`] = `
.c0 {
  grid-column: 1/4;
  width: 100%;
}

<div>
  <div
    class="c0"
  >
    <div>
      GridFluid
    </div>
  </div>
</div>
`;
